### PR TITLE
Update mixer deploy scripts

### PR DESCRIPTION
**Pull request by shifucun**
- Move docker build script to reside in the same folder as Dockerfile.
- Make a new script to build and push binaries from local code

### CODE REVIEW

File: build/push_go_protoc_image.sh [hunk 1]
```diff
@@ -15,5 +15,10 @@
 
 set -e
 
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```
Comment (pradh) [inline build/push_go_protoc_image.sh:18]: A comment on what it does?

**Review by pradh — APPROVED** [review]
Thanks!

File: scripts/push_binary.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
+  #!/bin/bash
```
Comment (beets) [inline scripts/push_binary.sh:1]: nit: spacing

**Review by beets — APPROVED** [review]
thanks for this!

File: build/push_go_protoc_image.sh (hunk 1 above)
Comment (shifucun) [reply build/push_go_protoc_image.sh:19]: Done

File: scripts/push_binary.sh (hunk 2 above)
Comment (shifucun) [reply scripts/push_binary.sh]: Done

**Review by shifucun — COMMENTED** [review]
thanks for review!

**Pull request merged**